qualysapi/version.py

```diff
@@ -1,3 +1,3 @@
 __author__ = 'Parag Baxi <parag.baxi@gmail.com>'
 __pkgname__ = 'qualysapi'
-__version__ = '4.1.0'
\ No newline at end of file
+__version__ = '5.0.3'
```

Comment:
Here maybe I would change the info or add some more to make sure that people don't get confused because our fork and the original library.